src/test/resources/TestNG.xml

```diff
@@ -2,11 +2,15 @@
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
 <suite name="TestNG">
-
+    <parameter name="baseURL" value="https://bbb.testpro.io/"/>
     <test name="Example Test" preserve-order="false">
         <classes>
-            <class name="LoginTests"/>
             <class name="BaseTest"/>
+            <class name="Homework21">
+                <methods>
+                    <include name="renamePlaylist"/>
+                </methods>
+            </class>
         </classes>
     </test>
 </suite>
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/225553006-03624768-4963-4578-8ce8-0c78ebd0621e.png)
![image](https://user-images.githubusercontent.com/104761482/225553392-e7d53308-33ee-4c5a-8907-90651f476cb0.png)

Hint

Explore test results

![Test status](https://user-images.githubusercontent.com/104761482/225553109-bfeec8e1-053b-4c14-b88e-51e40e6f05f4.png)



![Test error_](https://user-images.githubusercontent.com/104761482/225553141-aed94b3b-db85-40f9-a26c-460f303b9f44.png)